README.md

```diff
@@ -1,44 +1,82 @@
+Ada Lovelace is starting a pet rescue service advertising pets in need of rescue online.  This App should load the list of pets automatically and present a list of pets with thumbnail images.  The user should be able to select a pet and see details, add new pets and remove a pet from the App once adopted.
 
-### `npm run build`
+### Setup
 
-Builds the app for production to the `build` folder.<br>
-It correctly bundles React in production mode and optimizes the build for the best performance.
+1. Fork and clone this repository
+2. run `npm install` to install dependencies
+3. run `npm start` to start the dev server
 
-The build is minified and the filenames include the hashes.<br>
-Your app is ready to be deployed!
+## Requirements
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+Each of the waves includes a number of questions. Though you should write your own code, feel free to collaborate with other students as you work on these questions.
 
-### `npm run eject`
+### Wave 0: Reading
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+We have already implemented some parts of this application:
 
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+- Code to load JSON pet data in `App.js`
+- 3 functional stateless components: `PetList`, `PetCard` and `PetDetails`
+- CSS for all components. You shouldn't need to write any CSS for this project.
+  - All of our CSS uses [BEM naming](http://getbem.com/naming/)
 
-Instead, it will copy all the configuration files and the transitive dependencies (Webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+Before you start writing code, read through what's already here and make sure you understand it by answering the following questions:
 
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
+- How do the components relate to each other? Draw a diagram.
+- How does data get from `App.js` to `PetCard.js`?
 
-## Learn More
+### Wave 1: Viewing Details
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+When the user clicks the `select` button the app should present the user with additional details on the selected pet.  Read through the provided `propTypes` attribute of the given components, **what information does that provide**?
```

Comment:
Is there only one selected pet at a time? Could I hit "select" on all of the pets so that all of them have their details showing?

Reply:
I just saw the demo. Could we call this "Expand Details" instead of "select" or something similar?